src/app/app-routing.module.ts

```diff
@@ -1,30 +1,38 @@
-import {NgModule} from '@angular/core';
-import {Routes, RouterModule} from '@angular/router';
-import {AuthenticationGuard} from 'microsoft-adal-angular6';
-import {LandingPageComponent} from '@app/landing-page/landing-page.component';
-import {TechnicalRecordCreateComponent} from '@app/technical-record-create/technical-record-create.component';
-import {TestRecordComponent} from '@app/test-record/test-record.component';
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { AuthenticationGuard } from 'microsoft-adal-angular6';
+import { LandingPageComponent } from '@app/landing-page/landing-page.component';
+import { TechnicalRecordCreateComponent } from '@app/technical-record-create/technical-record-create.component';
+import { LoadPreparersGuard } from '@app/store/guards/load-preparers.guard';
+import { LoadTestStationsGuard } from '@app/store/guards/load-test-stations.guard';
 
 const routes: Routes = [
-  {path: '', component: LandingPageComponent, canActivate: [AuthenticationGuard]},
+  { path: '', component: LandingPageComponent, canActivate: [AuthenticationGuard] },
   {
     path: 'search',
-    loadChildren: './technical-record-search/technical-record-search.module#TechnicalRecordSearchModule',
+    loadChildren:
+      './technical-record-search/technical-record-search.module#TechnicalRecordSearchModule',
+    canActivate: [AuthenticationGuard]
+  },
+  {
+    path: 'create',
+    component: TechnicalRecordCreateComponent,
     canActivate: [AuthenticationGuard]
   },
-  {path: 'create', component: TechnicalRecordCreateComponent, canActivate: [AuthenticationGuard]},
   {
     path: 'technical-record',
     loadChildren: './technical-record/technical-record.module#TechnicalRecordModule',
     canActivate: [AuthenticationGuard]
   },
-  {path: 'test-record/:id', component: TestRecordComponent, canActivate: [AuthenticationGuard]},
-
+  {
+    path: 'test-record/:id',
+    loadChildren: './test-record/test-record.module#TestRecordModule',
+    canActivate: [LoadPreparersGuard, LoadTestStationsGuard]
```

Comment:
Your class names should be PreparersGuard and TestStationsGuard. It is your action names that should be LoadPreparersGuard, LoadPreparersGuardSuccess, LoadTestStationsGuard, LoadTestStationsGuardSuccess